va/va_enc_hevc.h

```diff
@@ -45,6 +45,200 @@ extern "C" {
  * @{
  */
 
+/** Attribute value for VAConfigAttribEncHEVCFeatures.
+ *
+ * This attribute decribes the supported features of an HEVC/H.265
+ * encoder configuration.
+ *
+ * All of the field values in this attribute are VA_FEATURE_* values,
+ * indicating support for the corresponding feature.
+ */
+typedef union VAConfigAttribValEncHEVCFeatures {
+    struct {
+        /** Separate colour planes.
+         *
+         * Allows setting separate_colour_plane_flag in the SPS.
+         */
+        uint32_t separate_colour_planes     : 2;
+        /** Scaling lists.
+         *
+         * Allows scaling_list() elements to be present in both the SPS
+         * and the PPS.  The decoded form of the scaling lists must also
+         * be supplied in a VAQMatrixBufferHEVC buffer when scaling lists
+         * are enabled.
+         */
+        uint32_t scaling_lists              : 2;
+        /** Asymmetric motion partitions.
+         *
+         * Allows setting amp_enabled_flag in the SPS.
+         */
+        uint32_t amp                        : 2;
```

Comment:
If amp is always supported by driver, how to set this field?

Reply:
If the driver requires amp to always be enabled then it should return VA_FEATURE_REQUIRED in this field.